rest/route/project.go

```diff
@@ -196,9 +196,11 @@ func (h *versionsGetHandler) Run(ctx context.Context) gimlet.Responder {
 // PATCH /rest/v2/projects/{project_id}
 
 type projectIDPatchHandler struct {
-	project string
-	body    []byte
-	user    *user.DBUser
+	project           string
+	user              *user.DBUser
+	newProjectRef     *dbModel.ProjectRef
+	originalProject   *dbModel.ProjectRef
+	requestProjectRef *model.APIProjectRef
```

Comment:
is the main reason to have both request and new because one is the db model and one is the API model? If so can you rename them so that it's more obvious?

Reply:
Yeah, it's because the requestProjectRef has fields that newProjectRef doesn't have (delete fields, mostly). Personally I prefer this naming convention, because it's not just that it's the APIProjectRef, it's that it's the actual verbatim request that the user gave.

Reply:
Actually you're probably right. I'm going to rename it here and also in the repo route.